salt/srv/pillar/site.j2

```diff
@@ -0,0 +1,54 @@
+#*****************************************
+# SITE & RELEASE DATA COLLECTION
+# Save file as /srv/pillar/site.j2
+# this is jinja not yaml!!
+# Set ip-address here using single quotes.
+#*****************************************
+{%- set sitedata = {} %}
+{%- do sitedata.update( {
+
+      'host_ipv4': grains.ipv4[-1],
+      'host_ipv6': grains.ipv6[-1],
+      'tgtBindIp': grains.ipv4[-1],
+      'db_host': grains.ipv4[-1],
+      'dns_host1': '8.8.8.8',
+      'dns_host2': '64.6.64.6',
+      'ipdomain': 'example.com',
+      'timezone': 'America/Denver',
+
+      'port_controller': '50040',
+      'port_dock': '50050',
+
+      'hotpot_release': 'v0.3.3',
```

Comment:
Also can you change the release version to `v0.4.0`, because we have published bali release, and then try it again

Reply:
Also a possible cause is maybe you forgot to copy `policy.conf` to `/etc/opensds/policy.conf`?

Reply:
Ahhhh - such a small detail but that must be the cause!!!!  

Do you mean policy.conf?  Or the policy.json mentioned in keystone.sh script?
```
cp "$OPENSDS_DIR/examples/policy.json" "$OPENSDS_CONFIG_DIR"
```
Thanks @leonwanghui I'll report back after checking all these.

Reply:
I cannot find an `examples\policy.json` file anywhere in Hotpot and Sushi tarballs.

I see `opensds-installer/conf/policy.json` file; is that the file?